### ISSUE TITLE
No .tscn color syntax on master

### ISSUE THREAD

**Issue by sandygk** [godotengine/godot-vscode-plugin#532]
### Godot version

4.2 beta 3

### VS Code version

1.84.2

### Godot Tools VS Code extension version

1.3.1 downloaded from (master branch)

### System information

windows 11

### Issue description

I don't get syntax coloring for my tscn files using the version from the master branch downloaded from the readme. The .gd files are working fine:

<img width="870" alt="image" src="https://github.com/godotengine/godot-vscode-plugin/assets/6596836/5152679d-362b-4aa6-aa55-3167d83ea210">

### Steps to reproduce

Open a .tscn with the master's  branch version of the extension and notice the lack of color syntax

**Comment by sandygk** [comment]
It works fine with the version from the VSCode Marketplace

**Comment by DaelonSuzuka** [comment]
Known issue, already fixed in #529.

**Comment by sandygk** [comment]
@DaelonSuzuka I am still seeing this issue in the PR branch, I downloaded it and packaged the extension, installed it and still seeing the issue. 

Also, shouldn't we close this issue once the PR is merged, and we made sure it was actually fixed, instead of when it's open? Since technically, the issue is still there, and the PR may not even get merged. It may also prevent other people from opening the same bug issue, since they may check only the open bug issues to see if it was already reported
